### ISSUE TITLE
Unable to resolve module AccessibilityInfo

### ISSUE THREAD

**Issue by ChandanCC** [GetStream/react-native-activity-feed#181]
- Both iOS and Android
- Using regular React Native
- react-native@0.63.4
- react-native-activity-feed@1.1.0
- getstream@7.2.8

**Describe the bug**
Unable to use react-native-activity-feed in the project
The project works fine if I comment the screen which is using react-native-activity-feed
Otherwise gives the error

**To Reproduce**
Basic installation procedure from docs and `npx react-native run-android` 

**Expected behavior**
Should show a screen with the basic example empty activity-feed

**Screenshots**
<img width="1440" alt="Screenshot 2021-05-05 at 6 37 41 PM" src="https://user-images.githubusercontent.com/32446002/117145937-4e65da00-add1-11eb-94ab-79b8d26aaafc.png">
<img width="427" alt="Screenshot 2021-05-05 at 6 40 39 PM" src="https://user-images.githubusercontent.com/32446002/117146030-663d5e00-add1-11eb-9121-ecce11341a7f.png">

**Comment by vishalnarkhede** [comment]
Hey @ChandanCC Have you tried clean install? I think that should solve this issue. 

```sh

rm -rf node_modules && rm yarn.lock
yarn;
npx react-native run-android
```

Its wierd that its pulling react-native from `react-native-activity-feeds/node_modules/react-native`. Are you trying to locally link the package?

**Comment by ChandanCC** [comment]
It sort of did help me solve the issue. Thanks a lot!
I did try a clean install before also...
But maybe my packages along with the new changes after adding activity-feed were creating some issues.

I believe `npm audit fix --force` helped it work this time...
Can't say the exact reason.

Thanks a lot for such a quick response!

**Comment by ChandanCC** [comment]
BTW are there any possibilities of adding typescript down the line??

**Comment by vishalnarkhede** [comment]
Yes, we do have a plan to introduce typescript, next quarter.

**Comment by badalkhatri0924** [comment]
@vishalnarkhede I am also facing the same issue.

We are using the following versions

"react": "16.13.1",
"react-native": "0.63.4",


Attaching the whole package.json as a gist

https://gist.github.com/badalkhatri0924/153f363b571605e0f07f8928254cbd7c

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/81486442/195574114-cd6b9b03-0e62-4d7a-95ca-23a8a24ab2ed.png">
